src/node/README.md

```diff
@@ -19,4 +19,4 @@ The [`src/node/`](./) directory is a new directory introduced in
 [#14978](https://github.com/bitcoin/bitcoin/pull/14978) and at the moment is
 sparsely populated. Eventually more substantial files like
 [`src/validation.cpp`](../validation.cpp) and
-[`src/txmempool.cpp`](../txmempool.cpp) might be moved there.
+[`src/txmempool.cpp`](../txmempool.cpp) may be moved there.
```

Comment:
small NIT, feel free to ignore don't want this to get stuck on an insignificant word choice.

While `may` can be used in a situation signifying possibility, it's mainly used in a context of signaling permission to do something. With the given context, I believe `might` is the better word.